exercises/grep/example.go

```diff
@@ -0,0 +1,102 @@
+package grep
+
+import (
+	"bufio"
+	"os"
+	"strconv"
+	"strings"
+)
+
+type grepOptions struct {
+	n_line_numbers      bool
+	l_file_only         bool
+	i_any_case          bool
+	v_invert            bool
+	x_match_entire_line bool
+	prefix_filenames    bool
```

Comment:
let's not prefer underscore since it's discouraged by golint

Reply:
ok